### DISCUSSION TITLE
Reopening PR's

### DISCUSSION THREAD

**Discussion by robgordon89** [renovatebot/renovate#17068] [Request Help]
Using WhiteSource Renovate hosted app on github.com I am running into an issue where PR's are reopening even when they are closed by a user.

When using hostRules for private GitHub repos, PRs and the dependency dashboard are opened with the username specified in hostRules, and so subsequent requests to lookup PRs and Issues results in nothing found and so duplicates are reopened.

My config:

```
{
  "extends": [
    "config:base"
  ],
  "hostRules": [
    {
      "matchHost": "github.com",
      "username": "ziltoidbot",
      "encrypted": {
        "password": "..."
      }
    }
  ]
}
```

**Comment by Gabriel-Ladzaretti** [comment]
Are the PR titles being changed?

**Comment by rarkins** [comment]
You must not override github.com credentials of the hosted bot or bad things happen.

If the private repos are within the same org and both are private, you should not need to do anything.

If they are within a different org then use a longer matching `matchHost` like `"matchHost": "https://api.github.com/repos/THEORG/"`

But this won't always work either due to some quirks with the GitHub API. Where/how/why are you looking up those repos? e.g. which manager and datasource?

**Reply by robgordon89** [reply]
hi, thanks for this information, this is likely what is causing so many issues then. Let me try and explain why we want to do this

The manager is regex and the datasource is normally GitHub release.

And yes the repositories are private and cross org :( (multiple orgs actually) so we have a user setup that has access to all and that is why we setup match host

**Reply by viceice** [reply]
you need to add `hostType=github-releases` to the rule, so you don't override the default app host rule

**Reply by robgordon89** [reply]
ohhh that's a good shout let me try that 👍

**Reply by robgordon89** [reply]
that's the one, that seems to work now, and I can enable the dependency dashboard as well 👍 thanks for your help @viceice and @rarkins

**Reply by robgordon89** [reply]
ok so this worked for some time and then I noticed it does not work cross org 

org1/repo1 - can get access to repo org1/dep1 without auth 
org2/repo1 - can't seem to get access to org1/dep1 even with auth setup 

This is a normal GitHub release but cross-organization, it is quite a common setup I think to separate projects by org

Anything I can try?

**Reply by rarkins** [reply]
The Renovate app tokens do not have permissions right now to access github packages, meaning any auth needs to come from a token you provision which overrides the default github.com auth which the app sets.

**Reply by robgordon89** [reply]
when you say "The Renovate app tokens do not have permissions right now " do you mean this is a bug that is being worked on I have found a few issues/discussions and even PRs that relate to this issue but not sure what the current status is of it?

**Reply by rarkins** [reply]
Unimplemented features are not bugs. See #17705 for a feature discussion and if you are able to do any research and move things forward there, it would get us closer.
